ansible/roles/test/files/ptftests/fg_ecmp_test.py

```diff
@@ -78,36 +82,41 @@ def setUp(self):
             raise Exception("required parameter 'exp_flow_count' is not present")
         self.exp_flow_count = self.test_params['exp_flow_count']
 
+        if 'dst_ip' not in self.test_params:
+            raise Exception("required parameter 'dst_ip' is not present")
```

Comment:
@anish-n pytest.fail ?

Reply:
This is in the ptf test, I don't believe this is using pytest and has the pytest.fail available. Regardless, the exception will be reported upto the calling pytest, and it will show the failure there. Hope that makes sense.